### ISSUE TITLE
With Foundry v10: Non-fatal error when creating new Quick Encounter

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#114]
1. Select two tokens
2. Press the fist icon
3. The Note icon is created, but the QE Sheet isn't popped up because of this error: 

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'contents')
    at new JournalSheet (foundry.js:64023)
    at Function.createFrom (QuickEncounter.js:637)
```

**Comment by spetzel2020** [comment]
This occurred because we were calling `new JournalSheet` with the `JournalEntryPage` not the `JournalEntry`

**Comment by spetzel2020** [comment]
Released in 1.1.0
